app/swagger/swagger/requests/profile.rb

```diff
@@ -1415,6 +1415,113 @@ class Profile
           end
         end
       end
+
+      swagger_path '/v0/profile/contacts' do
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::ForbiddenError
+
+          key :description, 'Create a Next-of-Kin or Emergency Contact'
+          key :tags, [:profile]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Contact attributes'
+            key :required, true
+
+            schema do
+              key :$ref, :Contacts
+            end
+          end
+
+          response 201 do
+            key :description, 'Contact created'
```

Comment:
Should this return the contact ID for reference? I don't think you will use it from the response, but could be useful for debugging and calling the other operations.

Reply:
The source system doesn't accept or return ids. However, rails uses ids by default when creating a route, so there's a bit of a mismatch in that regard.

Reply:
My thought on this is that we would need UI to support a multi-edit flow, such as a multi-page form with a confirmation step, to support editing multiple records. The [UI design](https://www.figma.com/design/uYMdNtQnrzq7OiDl0rPF9k/Profile---Personal-health-care-contacts?node-id=0-444&node-type=canvas) supports editing one contact at a time. Should we get this information in from of UX so they might make a decision about multi-edit?

Reply:
I think we can mention it, though I think their design was based on some prior work, so perhaps creating/editing one at a time is the preferred UX.